the-blue-alliance-ios/View Controllers/Match/MatchViewController.swift

```diff
@@ -52,5 +58,19 @@ class MatchViewController: MyTBAContainerViewController {
 
         Analytics.logEvent("match", parameters: ["match": match.key!])
     }
+    
+}
 
+extension MatchViewController: MatchSummaryViewDelegate {
+    
+    func teamPressed(teamNumber: Int) {
+        guard let event = match.event else { return }
+        
+        // get team key that matches the target teamNumber
+        guard let teamKey = match.teamKeys.first(where: { $0.teamNumber == "\(teamNumber)"}) else { return }
+        
+        let teamAtEventVC = TeamAtEventViewController(teamKey: teamKey, event: event, myTBA: myTBA, showDetailEvent: true, showDetailTeam: true, statusService: statusService, urlOpener: urlOpener, persistentContainer: persistentContainer, tbaKit: tbaKit, userDefaults: userDefaults)
```

Comment:
We need `showDetailEvent` and `showDetailTeam` shouldn't both be true here - we need to figure out which one we want. The right way to do this would be to figure out how we entered the view and show the opposite, but I think we'll just leave it to always push to the Event to keep things simpler.